project/FrgaalJavaCompiler.scala

```diff
@@ -29,8 +29,7 @@ object FrgaalJavaCompiler {
   val frgaal      = "org.frgaal" % "compiler" % "19.0.1" % "provided"
   val sourceLevel = "19"
 
-  val debugArg =
-    "-J-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=localhost:8000"
+  val debugArg = "-J" + org.enso.build.WithDebugCommand.DEBUG_OPTION
```

Comment:
Addresses this https://github.com/enso-org/enso/pull/7101/files#r1289974240 comment which I haven't probably posted before.

Reply:
Fine. If you insist. But note that if somebody sets `set javacOptions += FrgaalJavaCompiler.debugArg` inside an SBT project that invokes the compiler more than once, it will fail. Because `WithDebugCommand.DEBUG_OPTION` causes the started JVM to fail unless there is a debugger waiting for it to attach. I still don't like this. I think that debugging compilation is *very* different from debugging unit tests, even though, technically, they are both just JVM processes.

Reply:
> invokes the compiler more than once ... causes the started JVM to fail unless there is a debugger waiting for it to attach

I see.

Reply:
Undone in b5f5fddb3c